apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterPreviewContent.tsx

```diff
@@ -301,7 +302,7 @@ const NewsletterPreviewContent: React.FC<{
                                             )}
                                             style={{color: sectionTitleColor}}>Reimagining How We Eat</h3>
                                         <p className="mb-6" style={{color: textColor}}>Consumers are also starting to pay more attention. There&apos;s a growing movement toward mindful eating—not just in terms of ingredients, but in how we support the systems that bring food to our tables. Choosing to pick up instead of ordering in, tipping delivery drivers fairly, or subscribing to local restaurant coalitions can all make a difference.</p>
-                                        <p className="mb-6" style={{color: textColor}}>Ultimately, the story of delivery apps isn’t just about technology or convenience—it’s about the kind of communities we want to live in. And that future depends, in part, on the choices we make every day.</p>
+                                        <p className="mb-6" style={{color: textColor}}>Ultimately, the story of delivery apps isn&apos;t just about technology or convenience—it&apos;s about the kind of communities we want to live in. And that future depends, in part, on the choices we make every day.</p>
```

Comment:
Were these html entities added automatically or did you find bugs with the non-entity versions?